### ISSUE TITLE
expected times (ldrs500)

### ISSUE THREAD

**Issue by cmadland** [TWUOnline/bug-tracking#19]
**Which course contains the bug?**  
- LDRS 500 FAR Master
 
 
**Describe the error and/or your suggestion**  
Learning ACtivities need 'expected completion times'
 
 
**Location of the bug**  
Please provide a specific link to the page with the error, as well as where the error is on the page:
1. URL:  https://learn.twu.ca/course/view.php?id=10719&section=2
2. Unit # 2        Topic #  Overview       Activity#  all
 
**Optional: Screenshots**

**Comment by kmarjanovic** [comment]
I've added tentative times, but will ask Cathy to check the accuracy.
In the future, if a course is live like this one, we should either leave out missing components (times), or say an hour, which most activities take. We can confirm with SME.

**Comment by cmadland** [comment]
At first I thought that it’s fine because I was only changing the FAR MASTER site, and then I realized I’m an idiot.

Sent from my iPhone

On Sep 30, 2019, at 6:00 PM, Kelly Marjanovic <notifications@github.com> wrote:

﻿

I've added tentative times, but will ask Cathy to check the accuracy.
In the future, if a course is live like this one, we should either leave out missing components (times), or say an hour, which most activities take. We can confirm with SME.
